low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -0,0 +1,83 @@
+import os
+import shutil
+
+from cortx.sspl.bin.sspl_constants import (PRODUCT_FAMILY,
+ file_store_config_path, DATA_PATH)
+from cortx.utils.process import SimpleProcess
+from cortx.utils.service import Service
+from cortx.utils.conf_store import Conf
+from cortx.sspl.bin.error import SetupError
+
+SSPL_CONFIGURED=f"/var/{PRODUCT_FAMILY}/sspl/sspl-configured"
+
+class HardReset:
+    """Performs SSPL Hard reset.
+    'hard' is used to reset configs and clean log directory """
+
+    name = "hard"
+    user_present=False
+
+    def del_file(self, filename):
+        if os.path.exists(filename):
+            os.remove(filename)
+
+    def process(self):
+        # stop sspl service
+        Service('dbus').process('stop', 'sspl-ll.service')
+
+        # Remove sspl_conf
+        self.del_file(file_store_config_path)
+
+        # Remove sspl-configured file
+        self.del_file(SSPL_CONFIGURED)
+
+        # Remove sspl data
+        shutil.rmtree(DATA_PATH, ignore_errors=True)
+
+        # Remove sspl-ll user if preset
+        CMD = "id -u sspl-ll"
+        output, error, returncode = SimpleProcess(CMD).run()
+        if returncode != 0:
+            raise SetupError(returncode, error, CMD)
```

Comment:
Same as comment given on line: 49.
We won't see CMD in error message.

Reply:
same.

Reply:
same.